packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethodByCode.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
What do you think about leaving `braintree` here since it is a default offering from Venia?

Reply:
When we groomed this, it was mentioned that Braintree is only viable in the US based on huge fees and support, and therefor not a great default. This is by design 👍

Reply:
Got it